src/content/pages/account/login/index.tsx

```diff
@@ -87,9 +88,12 @@ const Login = () => {
  * Component for the login form.
  */
 const LoginForm = () => {
+    const navigate = useNavigate();
+
+    const {JWT, setJWT} = useContext(UserContext)
```

Comment:
Instead of calling `useContext` in each component, it is better to wrap this line into a  useAuthContext hook that is exported from the `AuthContext`.